docs/events.md

```diff
@@ -4,6 +4,24 @@ Events that TrenchBoot appeared on.
 
 ## Worldwide conferences
 
+* Xen Project Summit 2024
+    * Video: [Challenges and Status of Enabling TrenchBoot in Xen Hypervisor](https://www.youtube.com/watch?v=RVK52BCM-ZM)
+    * [Slides](slides/trenchboot_in_xen_2024.pdf)
+* FOSDEM 2024
+    * Video: [Trenchboot - project status update](https://video.fosdem.org/2024/aw1126/fosdem-2024-3724-trenchboot-project-status-update.av1.webm)
+    * [Slides (DRTM Status)](slides/DRTM-presentation-FOSDEM_2024.pdf)
+    * [Slides (Trenchboot AEM Status)](slides/TrenchBoot_AEM_Project_Status_fosdem_2024.pdf)
+* Xen Developer & Design Summit 2023
+    * Video: [Paving the Path for TrenchBoot DRTM in Xen Hypervisor](https://www.youtube.com/watch?v=A9GrlQsQc7Q?start=8152)
+    * [Slides](slides/Paving_the_Path_for_TrenchBoot_DRTM_in_Xen_Hypervisor.pdf)
+* Qubes OS Summit 2022
+    * Video: [TrenchBoot - The only AEM-way to boot Qubes OS](https://www.youtube.com/watch?v=A9GrlQsQc7Q&t=17440s)
+    * [Slides](slides/TrenchBoot-the-only-AEM-way-to-boot-Qubes-OS_2022.pdf)
+* Linux Plumbers Conference 2022
+    * Video: [TrenchBoot Update](https://www.youtube.com/watch?v=FFh3fWvVv0o)
+    * [Slides](slides/TrenchBoot%20-%20LPC%202022%20-%20Final.pdf)
+* FOSDEM 2021
+    * Video: [Secure Upgrades with DRTM](https://video.fosdem.org/2021/D.firmware/firmware_suwd.webm)
```

Comment:
I could not find the slides for this presentation.

Reply:
@dpsmith since this is your presentation, do you have those slides?

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/issues/40

Reply:
I trust Daniel will link them if still have them around, when finding some free time.